### ISSUE TITLE
How to integrate in Vue Js Application

### ISSUE THREAD

**Issue by Shilpa-K07** [flowrigami/flowrigami#1]
I'm trying to integrate this in my vue project. But getting "**Parsing error: Unexpected token**" in the file **Flowrigami.ts**.

![image](https://user-images.githubusercontent.com/70196456/132089113-df13b153-db1d-4f4f-a359-d1b4dc9948a9.png)

Please guide me how can I configure this in my vue js project.

Thanks

**Comment by flowrigami-sc** [comment]
Good day!
It's not enough information to understand the reason.
Possibly your project may not be using TypeScript, please check that you compile TS->JS before building your project.

**Comment by Shilpa-K07** [comment]
Yes, I compiled TS to JS. Able to reslove this error but having new problem with exports keyword.
![image](https://user-images.githubusercontent.com/70196456/132187843-c7571b97-5dd2-4c24-8f36-efe57b9d188c.png)

This is the tsconfig.json
{
    "compilerOptions": {
       // "module": "commonjs",
        "sourceMap": true,
        "baseUrl": "src",
        "paths": {
            "@app/*": ["src/*"]
          },
    },
    "exclude": [
        "node_modules"
    ]
}

**Comment by flowrigami-sc** [comment]
Hello @Shilpa-K07,

this kind of errors signal that smth is wrong in compilation/build process. "Exports" actually not found in DiagramApi.ts and Indicator.ts.
Maybe you can share how you compile and build your app?
